Issue/8641 update permission dialog

theck13 opened the pull request:
### Fix
Update the permission dialog shown when a new site is created as described in https://github.com/wordpress-mobile/WordPress-Android/issues/8641.  See the screenshots below for illustration.

![v2_sites_permission](https://user-images.githubusercontent.com/3827611/49157688-054afb00-f2e6-11e8-95e6-21096c237faa.png)

### Test
1. Go to ***Sites*** tab.
2. Tap ***Switch Site*** button.
3. Tap ***+*** action.
4. Tap ***Create WordPress.com site*** option.
5. Follow site creation steps.
6. Tap ***Configure*** button.
7. Notice ***Quick Start*** dialog is shown.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

SylvesterWilmott commented:
Looks awesome @theck13 

Aside from the image thing we already discussed, I've been speaking with Thomas and checking the Material guidelines and I think we should use a little more descriptive button labels:

Positive: `Accept`
Negative: `No Thanks`
Neutral: `Never`

theck13 commented:
I updated the dialog button text in https://github.com/wordpress-mobile/WordPress-Android/pull/8692/commits/8087c5049e1f051818a2b0c9af360ff5b16a7763.  I updated the dialog image in https://github.com/wordpress-mobile/WordPress-Android/pull/8692/commits/0b0c4f28b5aacc732dc03bb8d73a9da96d885727.

SylvesterWilmott commented:
Thanks @theck13
